docs/mkdocs/docs/technical/contributing.md

```diff
@@ -80,6 +80,35 @@ $MY_PYTHON -m pip install -ve .
 
 Note that as this will copy the binary to your Python installation this will have to be run after each and every change of a C++ file.
 
+mamba and conda-forge Quickstart
+================================
+
+ - [Install `mamba`](https://mamba.readthedocs.io/en/latest/installation.html)
```

Comment:
Can you add one line of context above - something like:

```
This quickstart builds a Conda release using build dependencies from Conda.
```

And then add the same for the Docker quickstart:

```
This quickstart builds a release using build dependencies from vcpkg. ArcticDB releases on PyPi use vcpkg dependencies in the manner as described below. 
```

Reply:
I have adapted both sections with 4bed36d03de01dc4d14f0657d8e6c181ec90c950.

Let me know if the wording conveys what you meant.